helloworld/protos/config.proto

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 Pants project contributors.
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+syntax = "proto2";
```

Comment:
Why not proto3?

Reply:
Their docs use proto2. I didn't know proto3 is a thing..Will update.